aten/src/ATen/cpu/vec256/vec256_float.h

```diff
@@ -6,6 +6,8 @@
 #include <sleef.h>
 #endif
 
+#include <iostream>
```

Comment:
this change can be removed (but if you dont want to meddle with this diff, then we can remove it after it's merged)

Reply:
Let's do it in a separate PR, otherwise it will be hard to traceback the syncing history in the future.

Reply:
@soumith Follow up here #7711